chromadb/api/__init__.py

```diff
@@ -23,8 +22,9 @@
     WhereDocument,
 )
 from chromadb.config import Component, Settings
-from chromadb.types import Database, Tenant
+from chromadb.types import Database, Tenant, Collection as CollectionModel
 import chromadb.utils.embedding_functions as ef
+from chromadb.api.models.Collection import Collection
 
 
 class BaseAPI(ABC):
```

Comment:
Split the relevant methods off of BaseAPI because they now take different arguments, and return different types.